sample_project/Assets/SampleViewer/Samples/OverviewMap/Scripts/LocationMarker.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright 2024 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+
+using Esri.ArcGISMapsSDK.Components;
+using Esri.ArcGISMapsSDK.Samples.Components;
+using Esri.ArcGISMapsSDK.Utils.GeoCoord;
+using Esri.GameEngine.Geometry;
+using UnityEngine;
+
+public class LocationMarker : MonoBehaviour
+{
+    [SerializeField] private ArcGISCameraComponent cameraComponent;
+
+    private ArcGISCameraControllerComponent cameraController;
+    private ArcGISLocationComponent cameraLocationComponent;
+    private ArcGISLocationComponent locationComponent;
+    private float northOffset = 225.0f;
```

Comment:
Why do we need to offset the rotation? Why 5/8?

Reply:
The orientation of the location marker is wrong. Without an offset it will be facing the wrong direction. 225 is the value it is off by. Only way to get rid of the offset is to fix it in a 3D modeling software like Blender

Reply:
if you're not planning to expose or change this, it would probably be better as `const`